test/acceptance/tests.py

```diff
@@ -923,6 +945,76 @@ def test_staff_override_at_beginning(self):
         ])
 
 
+class FullWorkflowRequiredTest(OpenAssessmentTest, FullWorkflowMixin):
```

Comment:
I'm going to try and refactor some of the common code out of this and `FullWorkflowOverrideTest`, as well as add some further assertions. I just wanted to get it functioning all the way through first.